pkgs/halo2-eth-membership-worker/src/worker.ts

```diff
@@ -37,24 +37,26 @@ export const halo2EcdsaWorker: IHalo2EthMembershipaWorker = {
     const isYOdd = calculateSigRecovery(v)
     const msgHash = hashMessage(message, 'bytes')
 
+    const params = await fetchKzgParams(k)
```

Comment:
if we happen to have `k=19`, this means downloading a file that is 65MiB.
That will definitely slow down the proving.
Is there a way to fetch this in advance?

Reply:
Good question, that should be figured out after benchmarking first.